lib/reporters/default.js

```diff
@@ -128,6 +131,10 @@ export default class Reporter {
 	}
 
 	startRun(plan) {
+		if (this.clearScreen) {
+			this.lineWriter.write(ansiEscapes.clearTerminal);
+		}
```

Comment:
Maybe move this down into the `if (this.watching && !plan.firstRun)` line below? It better connects `clearScreen` with watch mode plus maybe there's no need to clean the terminal on the first run? Or does that get weird?

Reply:
I had it somewhere down there to start, but then it occurred to me that `--clear-screen` *without* `--watch` had no effect at all, which simply felt "unhandled". Why *not* have it clear the screen? 🤔 

Since then I've been actually using it without the watch flag just as often as with it.